zcash_primitives/src/transaction/components/orchard.rs

```diff
@@ -126,7 +126,7 @@ pub fn read_cmx<R: Read>(mut reader: R) -> io::Result<ExtractedNoteCommitment> {
 pub fn read_note_ciphertext<R: Read>(mut reader: R) -> io::Result<TransmittedNoteCiphertext> {
     let mut tnc = TransmittedNoteCiphertext {
         epk_bytes: [0u8; 32],
-        enc_ciphertext: [0u8; 580],
+        enc_ciphertext: [0u8; 612],
```

Comment:
This is V3 specific at the moment, would need updating along with backward compatibility